nss-day-cohort-44/rare-rest-api-malicous-vortices
# Altered fixtures so that users do not begin subscibed to themselves. …

… Updated model so that ended_on field in subscriptions could be nullable.  Added custom action in users view to allow users to subscribe/unsubscribe to one another with a check to make sure they cannot subscripe to themselves.  Unsubscribing/resubscribing updates existing row in database, does not delete/recreate.

# ISSUE TICKET 
## Description


## Type of change


# Testing Instructions
`git fetch origin `
`git checkout ` 

Use URL path like: http://localhost:8000/users/2/subscribe

In Postman, try subscribing to yourself (replace 2 with current rare_user's primary key, this should not work), subscribing to another user, and unsubscribing/resubscribing to the user and monitoring the datetime field to see created_on/ended_on change.

ADDITION:  GET users and see if there are subscriptions attached.

User Instructions: 

# Checklist:
